integration_tests/mockApis/orchestration.ts

```diff
@@ -393,4 +393,17 @@ export default {
       },
     })
   },
+  stubGetPrison: ({ prisonCode }: { prisonCode: string }): SuperAgentRequest => {
```

Comment:
Might be worth setting a default here then you don't need to duplicate it everywhere you have `cy.task('stubGetPrison', { prisonCode: 'HEI' })` - i.e. that can just be `cy.task('stubGetPrison')`.

E.g. (untested) something like:

```suggestion
  stubGetPrison: ({ prisonCode = 'HEI' }: { prisonCode: string }): SuperAgentRequest => {
```